app.js

```diff
@@ -6,3 +8,20 @@
   // get user input
   // use user input to update state 
   // update DOM to reflect the new state
+
+
+import { champions } from './champions-data.js';
+import { renderChampion } from './render-champions.js';
+
+const championList = document.getElementById('champion-list');
+
+function displayChampions() {
+    championList.innerHtml = '';
+
+    for (let champion of champions) {
+        const championEl = renderChampion(champion);
+        championList.append(championEl);
+
```

Comment:
looping through array of data and passing each item to imported function to create elements from data and insert into document